### PR TITLE
SEOImages: User Story 1463880, Part 23

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031800008/Report/

**Comment by damabe** [comment]
Updated with feedback from other PR reviews. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019032600045/Report/

File: docs/framework/wpf/app-development/dialog-boxes-overview.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Standalone applications typically have a main window that both displays the main
 ## Message Boxes  
  A *message box* is a dialog box that can be used to display textual information and to allow users to make decisions with buttons. The following figure shows a message box that displays textual information, asks a question, and provides the user with three buttons to answer the question.  
   
- ![Word Processor dialog box](./media/dialogboxesoverviewfigure1.png "DialogBoxesOverviewFigure1")  
+ ![Screenshot that shows a Word Processor dialog box.](./media/dialog-boxes-overview/word-processor-dialog.png)
```
Comment (mairaw) [inline docs/framework/wpf/app-development/dialog-boxes-overview.md]:
```suggestion
 ![A Word Processor dialog box asking if you want to save the changes to the document before the application closes.](./media/dialog-boxes-overview/word-processor-dialog.png)  
```

File: docs/framework/wpf/app-development/dialog-boxes-overview.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Standalone applications typically have a main window that both displays the main
 ### Open File Dialog  
  The open file dialog box, shown in the following figure, is used by file opening functionality to retrieve the name of a file to open.  
   
- ![Open dialog box](./media/dialogboxesoverviewfigure2.png "DialogBoxesOverviewFigure2")  
+ ![Screenshot that shows an Open dialog box.](./media/dialog-boxes-overview/open-file-dialog-box.png)
```
Comment (mairaw) [inline docs/framework/wpf/app-development/dialog-boxes-overview.md]:
```suggestion
 ![An Open dialog box showing the location to retrieve the file.](./media/dialog-boxes-overview/open-file-dialog-box.png)  
```

File: docs/framework/wpf/app-development/dialog-boxes-overview.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ Standalone applications typically have a main window that both displays the main
 ### Save File Dialog Box  
  The save file dialog box, shown in the following figure, is used by file saving functionality to retrieve the name of a file to save.  
   
- ![Save As dialog box](./media/dialogboxesoverviewfigure3.png "DialogBoxesOverviewFigure3")  
+ ![Screenshot that shows a Save As dialog box.](./media/dialog-boxes-overview/save-file-dialog-box.png)
```
Comment (mairaw) [inline docs/framework/wpf/app-development/dialog-boxes-overview.md]:
```suggestion
 ![A Save As dialog box showing the location to save the file.](./media/dialog-boxes-overview/save-file-dialog-box.png)  
```

File: docs/framework/wpf/app-development/dialog-boxes-overview.md [hunk 4]
```diff
@@ -113,7 +113,7 @@ Standalone applications typically have a main window that both displays the main
 ### Creating a Modal Custom Dialog Box  
  This topic shows how to use <xref:System.Windows.Window> to create a typical modal dialog box implementation, using the `Margins` dialog box as an example (see [Dialog Box Sample](https://go.microsoft.com/fwlink/?LinkID=159984)). The `Margins` dialog box is shown in the following figure.  
   
- ![Margins dialog box](./media/dialogboxesoverviewfigure5.png "DialogBoxesOverviewFigure5")  
+ ![Screenshot that shows a Margins dialog box.](./media/dialog-boxes-overview/margin-size-dialog-box.png)
```
Comment (mairaw) [inline docs/framework/wpf/app-development/dialog-boxes-overview.md]:
```suggestion
 ![A Margins dialog box with fields to define left margin, top margin, right margin, and bottom margin.](./media/dialog-boxes-overview/margin-size-dialog-box.png)  
```

File: docs/framework/wpf/app-development/dialog-boxes-overview.md [hunk 5]
```diff
@@ -202,7 +202,7 @@ Standalone applications typically have a main window that both displays the main
   
  Once the validation rule is associated, [!INCLUDE[TLA2#tla_wpf](../../../../includes/tla2sharptla-wpf-md.md)] will automatically apply it when data is entered into the bound control. When a control contains invalid data, [!INCLUDE[TLA2#tla_wpf](../../../../includes/tla2sharptla-wpf-md.md)] will display a red border around the invalid control, as shown in the following figure.  
   
- ![Invalid left margin](./media/dialogboxesoverviewfigure7.png "DialogBoxesOverviewFigure7")  
+ ![Screenshot that shows an invalid left margin.](./media/dialog-boxes-overview/invalid-left-margin-dialog.png)
```
Comment (mairaw) [inline docs/framework/wpf/app-development/dialog-boxes-overview.md]:
```suggestion
 ![A Margins dialog box with a red border around the invalid left margin value.](./media/dialog-boxes-overview/invalid-left-margin-dialog.png)  
```

File: docs/visual-basic/programming-guide/language-features/arrays/array-dimensions.md [hunk 6]
```diff
@@ -21,14 +21,11 @@ A *dimension* is a direction in which you can vary the specification of an array
   
  The following illustrations show the conceptual structure of arrays with different ranks. Each element in the illustrations shows the index values that access it. For example, you can access the first element of the second row of the two-dimensional array by specifying indexes `(1, 0)`.  
   
- ![Graphic diagram of one&#45;dimensional array](../../../../visual-basic/programming-guide/language-features/arrays/media/arrayexdimone.gif "ArrayExDimOne")  
-One-dimensional array  
+ ![Diagram that shows a one&#45;dimensional array.](./media/array-dimensions/one-dimensional-array.gif)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/arrays/array-dimensions.md]:
```suggestion
 ![Diagram that shows a one-dimensional array.](./media/array-dimensions/one-dimensional-array.gif)  
```

File: docs/visual-basic/programming-guide/language-features/arrays/array-dimensions.md [hunk 7]
```diff
@@ -21,14 +21,11 @@ A *dimension* is a direction in which you can vary the specification of an array
   
  The following illustrations show the conceptual structure of arrays with different ranks. Each element in the illustrations shows the index values that access it. For example, you can access the first element of the second row of the two-dimensional array by specifying indexes `(1, 0)`.  
   
- ![Graphic diagram of one&#45;dimensional array](../../../../visual-basic/programming-guide/language-features/arrays/media/arrayexdimone.gif "ArrayExDimOne")  
-One-dimensional array  
+ ![Diagram that shows a one&#45;dimensional array.](./media/array-dimensions/one-dimensional-array.gif)  
   
- ![Graphic diagram of two&#45;dimensional array](../../../../visual-basic/programming-guide/language-features/arrays/media/arrayexdimtwo.gif "ArrayExDimTwo")  
-Two-dimensional array  
+ ![Diagram that shows a two&#45;dimensional array.](./media/array-dimensions/two-dimensional-array.gif)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/arrays/array-dimensions.md]:
```suggestion
 ![Diagram that shows a two-dimensional array.](./media/array-dimensions/two-dimensional-array.gif)  
```

File: docs/visual-basic/programming-guide/language-features/arrays/array-dimensions.md [hunk 8]
```diff
@@ -21,14 +21,11 @@ A *dimension* is a direction in which you can vary the specification of an array
...
-One-dimensional array  
+ ![Diagram that shows a one&#45;dimensional array.](./media/array-dimensions/one-dimensional-array.gif)  
   
- ![Graphic diagram of two&#45;dimensional array](../../../../visual-basic/programming-guide/language-features/arrays/media/arrayexdimtwo.gif "ArrayExDimTwo")  
-Two-dimensional array  
+ ![Diagram that shows a two&#45;dimensional array.](./media/array-dimensions/two-dimensional-array.gif)  
   
- ![Graphic diagram of three&#45;dimensional array](../../../../visual-basic/programming-guide/language-features/arrays/media/arrayexdimthree.gif "ArrayExDimThree")  
-Three-dimensional array  
+ ![Diagram that shows a three&#45;dimensional array.](./media/array-dimensions/three-dimensional-array.gif)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/arrays/array-dimensions.md]:
```suggestion
 ![Diagram that shows a three-dimensional array.](./media/array-dimensions/three-dimensional-array.gif)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good @damabe. Left some comments for you re. expanding a bit more on the alt text.

**Comment by damabe** [comment]
@mairaw Made the requested updates. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032800007/Report/

A couple of the alt text changes you requested are over 70 characters so will end up throwing an error on max alt length. I made the changes so please let me know if you want me to change to something else.

**Review by mairaw — APPROVED** [review]
Changes look good @damabe. Thank you!

**Pull request merged**